app/src/main/java/com/example/memeitupapp/util/MemeViewModelFactory.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.memeitupapp.util
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+
+object MemeViewModelFactory {
```

Comment:
Do you nide to declare this funtion inside an object?

Reply:
I think that it is right Lucho, because he use as a "utils object" and use right the inlne function

Reply:
you both are right 😉